test/interop/C/exportArray/arrayOpaquePointer_noArgDom.chpl

```diff
@@ -0,0 +1,24 @@
+use BlockDist;
+const D = {1..5} dmapped Block({1..5});
+
+export proc makeBlockArray(): [D] int {
+  var x: [D] int;
+  return x;
+}
+
+// Will generate a different signature than the return type for the previous
+// function
+export proc printBlock(x: [] int) {
```

Comment:
This seems like a TODO. Should this test be a future rather than a .notest file?